src/TipBox.vue

```diff
@@ -80,10 +85,16 @@
         return this.type === 'none'
       },
       isSeamless() {
-        return toBoolean(this.seamless);
+        return !this.light && this.seamless;
+      },
+      verticalDividerBool() {
+        return this.isSeamless && !this.headerBool;
+      },
+      horizontalDividerBool() {
+          return this.isSeamless && this.headerBool;
       },
-      headerContent() {
-        return this.header || this.heading;
+      headerBool() {
+        return this.$slots._header;
```

Comment:
`this.$slots._header` seems to be an object.  `headerBool` feels like a misnomer here. I think we should either keep it as `headerContent`, or make it return a boolean.

Reply:
Also, we seem to have two different naming conventions for this component: `isXXX` vs `xxxBool`. Should we stick to one of them?

Reply:
> Also, we seem to have two different naming conventions for this component: `isXXX` vs `xxxBool`. Should we stick to one of them?

Hmm, I followed the convention in `panelBase.js` for this, as its a little strange to call it `isHeader`. I could change it to `hasHeader` if preferred though!

Reply:
> If it's not a lot of effort, it would be great to add that in before we merge that in 🙂

I can't comment in the above comment for this strangely 😕 

I removed `toBoolean` as our docs dosen't show using `light/seamless="false"`, and
`this.light` is also used directly for `lightStyle` ( `light="false"` = truthy )

Reply:
> `this.$slots._header` seems to be an object. `headerBool` feels like a misnomer here. I think we should either keep it as `headerContent`, or make it return a boolean.

fixed 👍